cactusbot/commands/magic/alias.py

```diff
@@ -31,3 +31,19 @@ class Alias(Command):
             return "Alias {} removed.".format(alias)
         elif response.status == 404:
             return "Alias {} doesn't exist!".format(alias)
+
+    @Command.command("list", role="moderator")
+    async def list_aliases(self):
+        """List all aliases."""
+        response = await self.api.get_command()
+
+        if response.status == 200:
+            commands = (await response.json())["data"]
```

Comment:
If we got `200`, it should be.

Also, in the case that something weird happens and it *isn't* a key, the bot won't crash! This is run from within a handler, which means that it's run in a `try`/`except` - errors get logged, and the bot continues running.